vector/src/main/java/im/vector/activity/VectorCallViewActivity.java

```diff
@@ -1327,5 +1337,42 @@ public void endCall() {
         finish();
     }
 
+    //==============================================================================================================
+    // Manage call when the phone is locked
+    //==============================================================================================================
+
+    // Needed to let you answer call when phone is locked
+    private void turnScreenOnAndKeyguardOff() {
+        Log.d(LOG_TAG, "## VOIP turnScreenOnAndKeyguardOff");
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O_MR1) {
+            setShowWhenLocked(true);
+            setTurnScreenOn(true);
+            KeyguardManager keyguardManager = (KeyguardManager) getSystemService(Context.KEYGUARD_SERVICE);
+            keyguardManager.requestDismissKeyguard(this, null);
```

Comment:
Why are you doing this twice?
If I'm correct you're doing it at line 1362

Reply:
Yes I missed to remove these lines.